boa/environment.py

```diff
@@ -109,6 +109,40 @@ def patch_opcode(opcode_value, fn):
 _precompiles = {}
 
 
+def extract_arg_types(signature: str) -> str:
+    # Extract argument types and their names using regex
+
+    args = signature.split("(")[1].split(")")[0].split(",")
+    cleaned_args = [x.split(":")[1].strip().split("[")[0].lower() for x in args]
+
+    # Combine the cleaned argument types and return them in parentheses
+    return f"({', '.join(cleaned_args)})"
+
+
+def precompile(signature: str):
```

Comment:
i think this and the helper function should probably go in a new file to avoid bloating this too much. maybe `boa/precompiles.py`